redocly.yaml

```diff
@@ -0,0 +1,24 @@
+# NOTE: Only supports options marked as "Supported in Redoc CE"
+# See https://redocly.com/docs/cli/configuration/ for more information.
+
+extends:
+  - recommended
+
+rules:
+  no-unused-components: error
+
+theme:
+  # See https://redocly.com/docs/api-reference-docs/configuration/functionality/
+  openapi:
+    # disableSearch: true
+    # See https://redocly.com/docs/api-reference-docs/configuration/theming/
+    theme:
+      colors:
+        primary:
+          main: "#c54302"
+      typography:
+        fontFamily: "IBM Plex Sans, -apple-system, blinkmacsystemfont, Segoe UI, roboto, oxygen-sans, ubuntu, cantarell, Helvetica Neue, sans-serif;"
+      sidebar:
+        activeTextColor: "#c54302"
+        groupItems:
+          fontWeight: bold
```

Comment:
We can do _some_ customization here. Anything [on this page](https://redocly.com/docs/api-reference-docs/configuration/theming/) marked with "Supported in Redoc CE".